docs/source/dummies_node.rst

```diff
@@ -0,0 +1,650 @@
+
+	class Node(Base, SharedMixin):
+
+Immediately below this you'll see
+::
+
+	__tablename__ = "node"
+
+And as you might have guessed this specifies that objects of this class (Node) will get stored in a table called node. The next few lines create some columns that you should recognize from Postico:
+::
+
+    #: A String giving the name of the class. Defaults to
+    #: ``node``. This allows subclassing.
+
+    type = Column(String(50))
+    __mapper_args__ = {
+        'polymorphic_on': type,
+        'polymorphic_identity': 'node'
+    }
+
+    #: the id of the network that this node is a part of
+    network_id = Column(Integer, ForeignKey('network.id'), index=True)
+
+    #: the network the node is in
+    network = relationship(Network, backref="all_nodes")
+
+    #: the id of the participant whose node this is
+    participant_id = Column(Integer, ForeignKey('participant.id'), index=True)
+
+    #: the participant the node is associated with
+    participant = relationship(Participant, backref='all_nodes')
+
+Let's go through these one at a time. The first one creates a column called type, and specifies that its a String up to 50 characters long. The immediately following lines allow this column to take on different values (i.e. be "polymorphic"). Why would we want this? Well, remember that due to table/object duality at some point any row in the table is going to be read and turned into an object, but the program needs to know what kind of object to turn in into. We might expect things in the Node table to be turned into Nodes, and in general you are right. But remember how we discussed above that different kinds of Nodes can be created (bots, for instance). The type column is what let's the program know what kind of Node object to turn each row of the table into. We'll see example where types other than "node" are used later on, but for now, let's just stick with "node".
+
+The next row creates the network_id column. It contains an integer (not a String). The next bit (`ForeignKey('network.id')`) might seem a bit mysterious, but again it can be solved by thinking about row/object duality. Recall that if you want to know details of a nodes network you can just do something like:
+::
+
+	node.network.creation_time
+
+well this ForeignKey is what let's you do that. It says that whatever you get by typing `node.network_id`, you should get the same thing by typing `node.network.id`. In fact it doesn't just say that you *should* get the same thing, it says you *will* get the same thing. This is just firmly cementing the relationship between a node and its network.
+
+The final bit (`index=True`) really doesn't need to trouble you. It's basically an instruction to the database to keep tabs on this column. In practice it slightly slows down row creation, but hugely speeds up searches across the table. If we didn't have this set to true experiments would steadily slow down as more participants took part in the experiment and the tables grew.
```

Comment:
"It's basically an instruction to the database to keep tabs on this column."
An unintentional pun here?